cmd/editorconfig-checker/main_test.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"bytes"
+	"os"
+	"runtime"
+	"strings"
+	"testing"
+)
+
+var mainHasRun chan int
+
+func captureReturncode(code int) {
```

Comment:
```suggestion
func captureReturnCode(code int) {
```

Reply:
hm.. I never knew I treated "returncode" as a single word.. although I guess proper english would write that as "return code"... Anyway, fixed it.